$OpenBSD: patch-libaccounts-glib_Makefile_am,v 1.1 2017/06/03 10:09:10 espie Exp $
this option is deadly, you have to provide a full list of libraries,
which this port doesn't (missing at least libc).

Index: libaccounts-glib/Makefile.am
--- libaccounts-glib/Makefile.am.orig
+++ libaccounts-glib/Makefile.am
@@ -15,7 +15,6 @@ libaccounts_glib_la_LIBADD = \
 libaccounts_glib_la_LDFLAGS = \
 	$(COVERAGE_LDFLAGS) \
 	-version-info 1:3:1 \
-	--no-allow-shlib-undefined \
 	-export-symbols-regex '^ag_'
 
 BUILT_SOURCES = \
